src/renderer/services/formatting.js

```diff
@@ -32,6 +32,14 @@ const formatNumberBase = (value, wholeNumberFormat) => {
 };
 
 export default {
+  abbreviateNumber(value, wholeNumberFormat = formats.WHOLE_NUMBER, defaultValue = 'N/A') {
```

Comment:
Can you please put a unit test placeholder for this in `formatting.spec.js`? Tests on dev are currently broken, so I will fix and implement proper tests on a branch I have going. Thanks.

Reply:
Good call.